ibis/expr/types/generic.py

```diff
@@ -2480,6 +2481,33 @@ def literal(value: Any, type: dt.DataType | str | None = None) -> Scalar:
     Traceback (most recent call last):
       ...
     TypeError: Value 'foobar' cannot be safely coerced to int64
+
+    Literals can also be used in a deferred context.
+
+    Here's an example of constructing a table of a column's type repeated for
+    every row:
+
+    >>> from ibis import _, selectors as s
+    >>> ibis.options.interactive = True
+    >>> t = ibis.examples.penguins.fetch()
+    >>> t.select(s.across(s.all(), ibis.literal(_.type(), type=str).name(_.get_name())))
```

Comment:
This is kind of (IMO) an odd use case, and it takes up a bunch of vertical docstring space (all rows have the same value). Any way you could limit the output (`.limit(1)` perhaps) a bit just to improve the ratio of `good-use-of-literal:odd-use-of-literal` in this doc :)?

Reply:
Yeah, agree.

Reply:
Done.